README.md

```diff
@@ -110,15 +110,53 @@ namespace FileUploader
 ```
 
 ## Running Minio Client Examples
+#### On Windows 
 * Clone this repository.
 
-* Build the project to produce the Minio.Examples console app.
+* Build the project in Visual Studio to produce the Minio.Examples console app.
 
-* Move into Minio.Examples directory and enter your credentials and bucket name, object name etc.
+* Enter your credentials and bucket name, object name etc.in Minio.Examples/Program.cs
   Uncomment the example test cases such as below in Program.cs to run an example.
 ```cs
   //Cases.MakeBucket.Run(minioClient, bucketName).Wait();
 ```
+* Run the Minio.Client.Examples.NET452 or Minio.Client.Examples.NetCore project from Visual Studio
+#### On Linux (Ubuntu 16.04 and above)
+
+##### Setting up Mono and .NETCore on Linux
+<blockquote> NOTE: minio-dotnet requires mono 5.0.1 stable release and .NET Core 1.0 SDK to build on Linux. </blockquote>
+```
+bash $ git clone https://github.com/minio/minio-dotnet && cd minio-dotnet 
```

Comment:
@harshavardhana , mono & .netcore install are now in a script. Addressed other comments as well.